server/protocols/webadmin-integration-test/distributed-webadmin-integration-test/src/test/java/org/apache/james/webadmin/integration/rabbitmq/ConsistencyTasksIntegrationTest.java

```diff
@@ -303,16 +336,65 @@ void shouldRecomputeQuotas(GuiceJamesServer server) throws Exception {
             .basePath(TasksRoutes.BASE)
             .get(taskId + "/await");
 
-        QuotaProbesImpl quotaProbe = server.getProbe(QuotaProbesImpl.class);
         assertThat(
-            quotaProbe.getMessageCountQuota(QuotaRoot.quotaRoot("#private&" + BOB.asString(), Optional.empty()))
-                .getUsed()
-                .asLong())
-            .isEqualTo(1);
+            imapMessageReader.getQuotaRoot(MailboxConstants.INBOX))
+            .startsWith(String.format("* QUOTAROOT \"INBOX\" #private&%s\r\n* QUOTA #private&%s (MESSAGE 1 50)",
```

Comment:
Let me suggest `.contains("* QUOTA #private&bob@domain.tld  (MESSAGE 1 50)")`

(no string concat in tests as it hurts raw value readability + assert only what we care about)